LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/client/LabGuestAccountHelper.java

```diff
@@ -0,0 +1,62 @@
+package com.microsoft.identity.labapi.utilities.client;
+
+import com.microsoft.identity.internal.test.labapi.model.ConfigInfo;
+import com.microsoft.identity.internal.test.labapi.model.UserInfo;
+import com.microsoft.identity.labapi.utilities.BuildConfig;
+import com.microsoft.identity.labapi.utilities.authentication.LabApiAuthenticationClient;
+import com.microsoft.identity.labapi.utilities.exception.LabApiException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class to facilitate loading guest accounts from the lab api
+ * Takes a lab config object and creates a {@link LabGuest} object to facilitate performing
+ * operations on guest accounts
+ */
+public class LabGuestAccountHelper {
+
+    public static LabGuest loadGuestAccountFromLab(final LabQuery query) {
+        final LabApiAuthenticationClient authenticationClient = new LabApiAuthenticationClient(
+                BuildConfig.LAB_CLIENT_SECRET
+        );
+        final LabClient labClient = new LabClient(authenticationClient);
```

Comment:
I don't think we want to do this here in this method. Probably construct the Lab Client in the constructor of this class.

Reply:
This is a static method though, there's no constructor currently for this class. Did you mean `LoadLabUserTestRule`?

Reply:
As an alternative, I'm thinking if all the methods in this class should move directly in the LabClient class

Reply:
and we get rid of this entire class

Reply:
That would work